py/server/deephaven/learn/__init__.py

```diff
@@ -71,7 +71,7 @@ def _validate(inputs: Input, outputs: Output, table: Table):
     input_columns_list = [input_.input.getColNames()[i] for input_ in inputs for i in
                           range(len(input_.input.getColNames()))]
     input_columns = set(input_columns_list)
-    table_columns = {col.name for col in table.columns}
+    table_columns = set(table.definition.keys())
```

Comment:
This is not order-preserving. Why is Python like this? `dict` is ordered, but `set` isn't?

Reply:
The previous set construction was not order preserving either. Looking at the code, the order doesn't matter, it's just after fast check (and I think intersection / difference).